reference/pkg/nodejs/@pulumi/cloud-aws/index.md

```diff
@@ -2589,7 +2589,7 @@ kind: VolumeKind;
 </h2>
 
 ```typescript
-let apiShape: "/Users/swgillespie/go/src/github.com/pulumi/pulumi-cloud/aws/node_modules/@pulumi/cloud/types" =  undefined as any;
+let apiShape: "/Users/luke/go/src/github.com/pulumi/pulumi-cloud/aws/node_modules/@pulumi/cloud/types" =  undefined as any;
```

Comment:
I wouldn't hold up this PR over it, but it'd be nice if the username of the person who ran the tool didn't show up here.

Reply:
Added this to https://github.com/pulumi/docs/issues/492.